app/models/validations/sales/financial_validations.rb

```diff
@@ -11,4 +11,12 @@ def validate_income1(record)
       end
     end
   end
+
+  def validate_grant(record)
+    return unless record.grant
+
+    unless (0..999_999).cover?(record.grant)
+      record.errors.add :grant, I18n.t("validations.financial.grant_out_of_range")
+    end
+  end
```

Comment:
Is this here only so that the error message could be different from the numeric validations?

Reply:
Yes I have been working on the assumption that the error message given in the ticket should be the error message displayed, but there seems to be some inconsistency on that form the tickets so I'm happy to take direction.
I assume you're asking as an error will get added anyway since I have applied @min and @max in the Question

Reply:
Asked a question about it in slack, cause I think @bibblobcode did something similar to get a different wording as well. Feels a bit redundant to run same validations twice